action-types.yml

```diff
@@ -0,0 +1,31 @@
+# See https://github.com/typesafegithub/github-actions-typing/
+inputs:
+  sudo:
+    type: boolean
+
+  install-dependencies:
+    type: boolean
+
+  limit-access-to-actor:
+    type: boolean
+
+  detached:
+    type: boolean
+
+  connect-timeout-seconds:
+    type: integer
+
+  tmate-server-host:
+    type: string
+
+  tmate-server-port:
+    type: integer
+
+  tmate-server-rsa-fingerprint:
+    type: string
```

Comment:
This is where I would wish for a much more powerful type declaration. RSA fingerprints have a _very_ specific format that could totally be automatically type-checked statically, but simply saying `string` won't help that.

Reply:
github-actions-typing supports just primitive types for now.

Reply:
Who knows, maybe in the future there will be some support for validations, like min and max values for numbers like `tmate-server-port`, or regexes for strings like `tmate-server-rsa-fingerprint`.

This is just not defined yet.
But actually it might be a good idea, so I created a feature request for it at https://github.com/typesafegithub/github-actions-typing/issues/205 :-)